README.md

```diff
@@ -1,5 +1,8 @@
-# Bonita Engine [![Build Status](https://travis-ci.org/bonitasoft/bonita-engine.svg)](https://travis-ci.org/bonitasoft/bonita-engine)
+# Bonita Engine
 
+[![Travis-ci Status](https://travis-ci.org/bonitasoft/bonita-engine.svg)](https://travis-ci.org/bonitasoft/bonita-engine)
```

Comment:
```suggestion
[![Travis-ci Status](https://travis-ci.org/bonitasoft/bonita-engine.svg)](https://travis-ci.org/bonitasoft/bonita-engine)
![Travis CI](https://img.shields.io/travis/bonitasoft/bonita-engine/master?label=Travis%20build&logo=travis)
```

Reply:
the following suggestion add a travis logo

![image](https://user-images.githubusercontent.com/27200110/65692075-416bfb00-e072-11e9-9aa4-11d27b731a0b.png)

Reply:
ok